lug-helper.sh

```diff
@@ -2278,9 +2313,9 @@ install_game() {
 
         # Run the appropriate installer
         if [ "$install_version" = "native" ]; then
-            lutris --install "$install_script" &
+            lutris && lutris --install "$install_script" &
         elif [ "$install_version" = "flatpak" ]; then
-            flatpak run --file-forwarding net.lutris.Lutris --install @@ "$install_script" @@ &
+            flatpak run net.lutris.Lutris && flatpak run --file-forwarding net.lutris.Lutris --install @@ "$install_script" @@ &
```

Comment:
this is meant to handle the vanishingly small edge case of users that have not opened lutris yet. some new users go right to the lug helper to install star citizen. Unfortunately `lutris -i` does not cause lutris to  populate its runtime and caches for features such as wine and dxvk. this results in an error message warning about missing version information. repeated attempts to install will fail until they open lutris separately from the lug helper install option.

`"Runtime Error('No path can be generated for DXVK because no version information is available.')"`

Reply:
I see, so that sounds a bit like a bug in Lutris that we should also report upstream?

Reply:
FYI, I just added this to the wiki here:
https://github.com/starcitizen-lug/knowledge-base/wiki/Troubleshooting#lutris-error-runtime-errorno-path-can-be-generated-for-dxvk-because-no-version-information-is-available